addons/core/translations/resources/en-us.yaml

```diff
@@ -348,6 +348,9 @@ credential-store:
       help: A PEM-encoded private key matching the client certificate from Client Certificate.
     client_certificate_key_hmac:
       label: Client Certificate Key HMAC
+    type:
```

Comment:
Take a look at the host-catalog resource to see how types and help are handled.  The names and descriptions of types are a bit more general and have utility beyond just forms.